dev-tools/mage/integtest.go

```diff
@@ -0,0 +1,236 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package mage
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"sync"
+
+	"github.com/pkg/errors"
+
+	"github.com/magefile/mage/mg"
+	"github.com/magefile/mage/sh"
+)
+
+const (
+	// BEATS_DOCKER_INTEG_TEST_ENV is used to indicate that we are inside of the
```

Comment:
Not a big fan of the abbreviation `INTEG` here but `INTEGRATION` is also pretty long.

Reply:
Yeah, I also don't like it much. One suggestion as valid as any other: maybe I'll remove the word `BEATS` and `ENV` because I feel like it's implicit in what we are doing. `DOCKER_INTEGRATION_TEST` doesn't seem so long.

Reply:
I definitely want to keep the `BEATS_` to make it clear that it originates from Beats if you see it somewhere. I can easily expand `INTEG` to `INTEGRATION`. This value is only used internally by the build system to identify "where" it's running so that it can execute the correct commands.